codec-http2/src/main/java/io/netty/handler/codec/http2/HttpConversionUtil.java

```diff
@@ -410,19 +412,30 @@ public static Http2Headers toHttp2Headers(HttpHeaders inHeaders, boolean validat
         return out;
     }
 
+    private static CharSequenceMap<AsciiString> toLowercaseMap(List<String> values) {
+        CharSequenceMap<AsciiString> result = new CharSequenceMap<AsciiString>();
```

Comment:
should we use the constructor that takes the `arraySizeHint` based on `values.size()` ?